powerstore/volume_helper.go

```diff
@@ -17,7 +17,7 @@ const (
 	TiB = 1024 * GiB
 )
 
-func updateVolState(volState *models.Volume, volResponse pstore.Volume, hostMapping []pstore.HostVolumeMapping, volPlan *models.Volume, operation string) {
+func updateVolState(volState *models.Volume, volResponse pstore.Volume, hostMapping []pstore.HostVolumeMapping, volGroupMapping pstore.VolumeGroups, volPlan *models.Volume, operation string) {
```

Comment:
as commented above
make this as method and
try to use https://github.com/dell/terraform-provider-powerstore/blob/dev/powerstore/defaults.go#L14